common/lib/common/kconfig.ts

```diff
@@ -389,7 +389,7 @@ const explorerCollectionUrl = (chainId: number, collAddress = ""): string => {
 
 // KREDEUM NFT URL
 const kredeumNftUrl = (chainId: number, nft: NftType): string =>
-  `./#/${getChainName(nft.chainId)}/${nft?.address}/${nft?.tokenID}`;
+  `#/${getChainName(nft.chainId)}/${nft?.address}/${nft?.tokenID}`;
```

Comment:
indicate why to change this ? this should cause problem with some direct links

Reply:
because on dapp it's working as a normal # target, and it permit to work on wordpress